install_goto.sh

```diff
@@ -8,7 +8,29 @@ touch ${HOME}/.goto/active-project
 ln -s /usr/local/opt/goto/bin/* /usr/local/bin/
 
 # add init_script to.bash_profile:
-echo "source start_goto" >> ${HOME}/.bash_profile
+
+echo "Goto needs to run a startup script for it to work. "
+echo "This can be automatically added now to your .bash_profile file"
```

Comment:
Skip this.. Redundant information. Go directly to the question.

Reply:
It depends a little on the target audience. Goto should be straightforward to use for the inexperienced terminal user also - and not knowing the internals of goto or bash - a user might be wondering why goto needs a start script, and what that will mean in practice

Reply:
Here is what I am suggesting

> Goto needs to run a startup script for it to work.
~This can be automatically added now to your .bash_profile file~    **# <-- Redundant**
Want to automatically add the goto startup script to your .bash_profile?

To make the first sentence redundant also just add **Required** at the end of the third
 
> ~Goto needs to run a startup script for it to work.~  **# <-- Redundant**
~This can be automatically added now to your .bash_profile file~    **# <-- Redundant**
Want to ~automatically~ add ~the~ goto startup script to ~your~ .bash_profile? **(Required)**

Also since you are running a script, specifying that something will happen automatically is sort of redundant, leaving us with this simple line

>Add  goto startup script to `.bash_profile`?  **(Required)**

Some observations :
1. No information is lost.
2. Fewer words makes any reader happy,  (experienced or inexperienced).
2. Expressing 1 step in 1 line is easier to understand.
3. An inexperienced user will read the sentence like this:
 >Something something is **required**

and just wants to click ENTER/OK.

4. An advanced user knows `.bash_profile` is being updated and can interrupt and do it manually, if he/she desires.